homeassistant/components/steam_online/config_flow.py

```diff
@@ -168,7 +173,14 @@ def get_accounts(self) -> list[dict[str, str | int]]:
         try:
             friends = interface.GetFriendList(steamid=self.entry.data[CONF_ACCOUNT])
             _users_str = [user["steamid"] for user in friends["friendslist"]["friends"]]
+            id_batches = list(_batch_ids(_users_str))
+            names = []
+            for id_batch in id_batches:
+                names.extend(
+                    interface.GetPlayerSummaries(steamids=id_batch)["response"][
```

Comment:
One thing to note here is that this is moving inside the `try/except`. Is it ok to capture and ignore errors with regarding user ids silently and without logging?